README.md

```diff
@@ -1,2 +1,27 @@
-# mysql-bundle
-A bundle for mysql and mysql-router machine charms
+# Canonical Distribution of MySQL + MySQLRouter
+
+Welcome to the Canonical Distribution of MySQL + MySQLRouter.
+
+The objective of this page is to provide directions to get up and running with Canonical MySQL charms.
+
+## Installation
+
+Currently, we support this distribution with Ubuntu 20.04.
+
+To get started, please install the following:
+- LXD
+- Juju
+
+Then, please bootstrap the juju controller with microk8s using `juju bootstrap localhost <controller_name>`.
+
+Finally add a juju model with `juju add-model <model-name>` and deploy the bundle with `juju deploy mysql-bundle`.
```

Comment:
I think we usually assume that users have a controller boostrapped. (I think we avoid giving instructions for install since a variety of clouds can be used with different configurations) 

```suggestion
## Usage
To deploy the bundle simply do:
```shell
juju deploy mysql-bundle
```
```

Reply:
Addressed in 7ec16b1340d88398e8af062ca014c4b739e291b8